requirements.txt

```diff
@@ -12,7 +12,7 @@ pyparsing==2.2.0
 pygments==2.2.0
 requests==2.20.0
 six==1.10.0
-sbp==2.4.5
+git+https://github.com/swift-nav/libsbp@4e8fc16b95639cd58649fe2d559230798f714e65#egg=sbp&subdirectory=python
```

Comment:
Reminder to replace this with a tag when the requisite libsbp versions is released.

Reply:
🔨